docs/modules/trino/pages/usage-guide/security.adoc

```diff
@@ -83,7 +83,7 @@ NOTE: Due to changes in the Trino OPA authorizer, rego rules differ between vers
 include::example$usage-guide/opa-bundle-trino-cm-414.yaml[]
 ----
 
-=== Rego for Trino version 428
+=== Rego for Trino version 428 and newer
```

Comment:
It would be great if we could re-write this policies to use Rego v1

Reply:
done https://github.com/stackabletech/trino-operator/pull/536/commits/6975b146966744223bdbcd484f07a1fa0ef84c1c